src/index.js

```diff
@@ -1,18 +1,13 @@
-/** @jsx Preact.h */
 import Preact from 'preact';
-import ErrorBox from './error';
-import App from './components/app';
+import { getScrollytellers } from './loader';
 
-let sections = () => {
-    __ODYSSEY__.utils.anchors.getSections('scrollyteller').forEach(section => {
-        let element = section.betweenNodes.shift().firstChild;
-        render(element, section);
-        section.substituteWith(element);
-    });
+let init = () => {
+    getScrollytellers().forEach(section => render(section.mountNode, section));
```

Comment:
I've changed the loader to now check if there is an `init-interactive` embed as the first child of the `#scrollyteller` and use it as the mount point (or create a new element if it's not there).